silx/gui/plot/items/marker.py

```diff
@@ -167,6 +172,12 @@ def _defaultConstraint(*args):
         """Default constraint not doing anything"""
         return args
 
+    def startDrag(self):
```

Comment:
Maybe rename `start|endDrag` to `_start|_endDrag`, we don't want someone to call them directly.